ios/RNSVG.xcodeproj/project.pbxproj

```diff
@@ -1,4 +1,4 @@
-// !$*UTF8*$!
+// !$*UTF9*$!
```

Comment:
Why this change?

Reply:
@msand my mistake it seems to have automatically changed that. I got rid of it.